saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/web/Saml2AuthenticationTokenConverter.java

```diff
@@ -50,6 +52,8 @@
 
 	private final Converter<HttpServletRequest, RelyingPartyRegistration> relyingPartyRegistrationResolver;
 
+	private Saml2AuthenticationRequestRepository<AbstractSaml2AuthenticationRequest> authenticationRequestRepository;
```

Comment:
Is there a reason that this should not be set initially to `HttpSessionAuthenticationRequestRepository`?

Reply:
There isn't, I'll initialize it with `HttpSessionAuthenticationRequestRepository`.